pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -110,12 +110,12 @@ in
   };
 
   noto-fonts-emoji = let
-    version = "2.028";
+    version = "2.034";
     emojiPythonEnv =
       python3.withPackages (p: with p; [ fonttools nototools ]);
   in stdenv.mkDerivation {
     pname = "noto-fonts-emoji";
-    version = builtins.replaceStrings [ "_" ] [ "." ] version;
```

Comment:
The last time a version had an underscore in it was 20th Sept 2020 `2020-09-16-unicode13_1` so I guess this can be dropped. But is there any point defining the version in the let section anymore?

Reply:
It doesn't really make a difference, so I'm just gonna leave it as is.

Reply:
sure, feel free to mark this resolved